src/components/urls/sourceFiles.ts

```diff
@@ -0,0 +1,7 @@
+const SourceFiles = {
```

Comment:
What are these?

Again, no absolute URLs. And especially none that are invalid.